package.json

```diff
@@ -107,6 +119,7 @@
     "verdaccio-theme"
   ],
   "scripts": {
+    "tslint": "tslint --project . --format verbose",
```

Comment:
About the scripts ... the other projects follow the same scripts, this provides a level of consistency across all projects. See here an example.

https://github.com/verdaccio/verdaccio-audit/blob/master/package.json#L10

```
"scripts": {
    "lint": "eslint . --ext .js,.ts",
    "test": "npm run lint && jest",
    "prepublish": "npm run build",
    "coverage:publish": "codecov",
    "commitmsg": "commitlint -e $GIT_PARAMS",
    "release": "standard-version -a -s",
    "type-check": "tsc --noEmit",
    "type-check:watch": "npm run type-check -- --watch",
    "build:types": "tsc --emitDeclarationOnly",
    "build:js": "babel src/ --out-dir lib --extensions \".ts,.tsx\" --source-maps inline",
    "build": "npm run build:types && npm run build:js"
  },
```

As you see, babel does the transpiling.